protocol/messenger_pairing_and_syncing_test.go

```diff
@@ -58,12 +59,111 @@ func (s *MessengerPairingSuite) TestEnableNonExistingInstallation() {
+	alice2, err := newMessengerWithKey(s.shh, s.privateKey, s.logger, nil)
+	s.Require().NoError(err)
+	defer TearDownMessenger(&s.Suite, alice2)
+
+	wrongTargetInstallationID := uuid.New().String()
+	mockRequest := requests.NewMockEnableInstallationAndPair(alice1.installationID, func() string {
+		return wrongTargetInstallationID
+	})
+	_, err = alice2.EnableInstallationAndPair(mockRequest)
+	s.Require().NoError(err)
+
+	_, err = WaitOnMessengerResponse(
+		alice1,
+		func(r *MessengerResponse) bool {
+			for _, i := range r.Installations() {
+				// We expect the installation to be added but no activity center notification
+				if i.ID == alice2.installationID && len(r.ActivityCenterNotifications()) == 0 {
+					return true
+				}
+			}
+			return false
+		},
+		"no messages",
+	)
+	s.Require().NoError(err)
+}
+
+// TestNewInstallationCreatedIsNotDeleted tests the scenario
+// 1. prepare AC NewInstallationCreated for alice2
+// 2. prepare AC NewInstallationCreated for alice3
+// 3. alice1.EnableInstallationAndSync
+// 4. check AC NewInstallationCreated for alice3 is not deleted
+func (s *MessengerPairingSuite) TestNewInstallationCreatedIsNotDeleted() {
+	alice1 := s.m
+	alice2, err := newMessengerWithKey(s.shh, s.privateKey, s.logger, nil)
+	s.Require().NoError(err)
+	alice3, err := newMessengerWithKey(s.shh, s.privateKey, s.logger, nil)
+	s.Require().NoError(err)
+	defer TearDownMessenger(&s.Suite, alice2)
+	defer TearDownMessenger(&s.Suite, alice3)
+
+	// prepare AC NewInstallationCreated for alice2
+	_, err = alice2.EnableInstallationAndPair(&requests.EnableInstallationAndPair{InstallationID: alice1.installationID})
+	s.Require().NoError(err)
+
+	// alice1 should get the installationID2 from alice2
+	s.expectInstallationAndACNotificationNewInstallationReceived(alice1, alice2.installationID)
+
+	// prepare AC NewInstallationCreated for alice3
+	_, err = alice3.EnableInstallationAndPair(&requests.EnableInstallationAndPair{InstallationID: alice1.installationID})
+	s.Require().NoError(err)
+
+	// alice1 should get the installationID3 from alice3
+	s.expectInstallationAndACNotificationNewInstallationReceived(alice1, alice3.installationID)
+
+	_, err = alice1.EnableInstallationAndSync(&requests.EnableInstallationAndSync{InstallationID: alice2.installationID})
+	s.Require().NoError(err)
+
+	_, err = WaitOnMessengerResponse(
+		alice3,
+		func(r *MessengerResponse) bool {
+			for _, i := range r.Installations() {
+				if i.ID == alice1.installationID {
+					return true
+				}
+			}
+			return false
+		},
+		"alice3 should get the installationID from alice1",
+	)
+	s.Require().NoError(err)
+
+	ac, err := alice3.ActivityCenterNotification(types.FromHex(alice1.installationID))
+	s.Require().NoError(err)
+	s.Require().Equal(ActivityCenterNotificationTypeNewInstallationCreated, ac.Type)
+	s.Require().False(ac.Deleted)
+}
+
+func (s *MessengerPairingSuite) expectInstallationAndACNotificationNewInstallationReceived(m *Messenger, installationID string) {
```

Comment:
I understand your intention, but this is a waaaaay too long function name 😄 
`expectInstallationReceived` is enough I think.